css.md

```diff
@@ -0,0 +1,171 @@
+# CSS & Less Guide
+
+This style guide aims to provide the ground rules for CFPB's CSS, such that it's highly readable and consistent across different developers on a team. 
+
+## Table of Contents
+
+1. [General Practices](#general-practices)
+1. [Property Order](#property-order)
+1. [Naming Conventions](#naming-conventions)
+1. [Property values](#property-values)
+1. [Media Queries](#media-queries)
+1. [Less](#less)
+    - [Less General Practices](#less-general-practices)
+    - [Selectors and Nesting](#selectors-and-nesting)
+    - [Variables](#variables)
+
+## General Practices
+
+- Use soft-tabs with a two space indent
+- One line per selector
+- Use a single space after the colon
+- Always end property declarations with a semicolon
+- Put spaces after `:` in property declarations
+- Put spaces before `{` in rule declarations
```

Comment:
Should add a bullet for `{` on the same line as the (last) selector, `}` on its own line.